.github/workflows/javadocs_scripts/parse_pom.py

```diff
@@ -0,0 +1,34 @@
+# Runs after `parse_versionstxt.py` and parses `google-cloud-bom/pom.xml` for the gapic-libraries-bom and handwritten library versions and updates those versions to `/site/data/variables.yaml`
+
+import xml.etree.ElementTree as ET
+import os.path
+
+namespaces = {
+    "m": "http://maven.apache.org/POM/4.0.0"
+}
+
+# List the dependencies to exclude from the variables.yaml file
+exclude_deps = ['google-java-format', 'google-cloud-bom']
```

Comment:
I've removed this part from the script. The only concern with extra entries in the variables.yaml file (renamed to `javaModulesVersions.yaml`) is that they're used to generate the main table on the site. However, I've kept the exclusion list in the `libraryTable_generation.py` script, so that should capture any entries to this file that we do not want to show up on the site.